test/Faker/Provider/de_CH/PersonTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Faker\Test\Provider\de_CH;
+
+use Faker\Calculator\Ean;
+use Faker\Generator;
+use Faker\Provider\de_CH\Person;
+use PHPUnit\Framework\TestCase;
+
+class PersonTest extends TestCase
+{
+    private $faker;
+
+    public function setUp()
+    {
+        $faker = new Generator();
+        $faker->addProvider(new Person($faker));
+        $this->faker = $faker;
+    }
+
+	public function testAvs13Number()
```

Comment:
indentation seems broken in this file, can you please check

Reply:
Done.